templates/test/dev/cluster-template-custom-builds.yaml

```
@@ -375,6 +384,39 @@ spec:
           New-Item -ItemType Directory -Force -Path C:\tmp\
         path: C:/create-temp-folder.ps1
         permissions: "0744"
+      - content: |
+          Stop-Service kubelet -Force
+
+          $$binaries=@("kubeadm", "kubectl", "kubelet", "kube-proxy")
+          $$ci_url="https://${AZURE_STORAGE_ACCOUNT}.blob.core.windows.net/${JOB_NAME}/${KUBE_GIT_VERSION}/bin/windows/amd64"
+          foreach ( $$binary in $$binaries )
+          {
+            echo "installing package: $$binary ${KUBE_GIT_VERSION}"
+            curl.exe --retry 10 --retry-delay 5 "$$ci_url/$$binary.exe" --output "c:/k/$$binary.exe"
+          }
+
+          $$CONTAINERD_URL="${WINDOWS_CONTAINERD_URL}"
+          if($$CONTAINERD_URL -ne ""){
+            Stop-Service containerd -Force
+            echo "downloading containerd: $$CONTAINERD_URL"
+            curl.exe --retry 10 --retry-delay 5 -L "$$CONTAINERD_URL" --output "c:/k/containerd.tar.gz"
+            cmd /c tar -zxvf c:/k/containerd.tar.gz -C "c:/Program Files/containerd" --strip-components 1
```

```suggestion
            tar.exe -zxvf c:/k/containerd.tar.gz -C "c:/Program Files/containerd" --strip-components 1
```

Just calling tar.exe should have the same effect as calling `tar` from cmd w/o starting extra processes

---

similar to curl.exe